main/ahrs.go

```diff
@@ -0,0 +1,237 @@
+package main
+
+import "math"
+
+var sampleFreq float64 = 512.0
+var beta float64 = 0.1 //values of 0.02 or 0.025 have also been suggested
+var q0, q1, q2, q3 float64 = 1.0, 0.0, 0.0, 0.0
+var attitudeX, attitudeY, attitudeZ float32
+

```

Comment:
attitudeX, attitudeY, and attitudeZ can be deleted and just do short var declarations, see below